lib/acts_as_paranoid/core.rb

```diff
@@ -36,7 +36,12 @@ def delete_all!(conditions = nil)
       end
 
       def delete_all(conditions = nil)
-        where(conditions).update_all(["#{paranoid_configuration[:column]} = ?", delete_now_value])
+        ar = where(conditions)
+        if paranoid_configuration[:touch]
+          ar.update_all(["#{paranoid_configuration[:column]} = ?, updated_at = ?", delete_now_value, Time.now])
```

Comment:
This code uses `delete_now_value` for the paranoid column, but uses `Time.now` for `updated_at` - we're assuming the format of the `updated_at` column, but retaining the old functionality for the paranoid column.